documentation/library_guide/parallel_api/range_based_api.rst

```diff
@@ -1,15 +1,107 @@
 A range adaptor, from the |onedpl_long| (|onedpl_short|) perspective, is a utility that transforms the base range,
 or another adapted range, into a view with custom behavior.
 
+|onedpl_short| only supports random access ranges, because they allow efficient and constant-time access to elements at any position in the range. This enables effective workload distribution among multiple threads or processing units, which is crucial for achieving high performance in parallel execution.
+
+|onedpl_short| introduces two sets of range based algorithms:
+
+* The oneapi::dpl::ext::ranges namespace supports integration with the Ranges Library coming with C++20 standard and introduced by std::ranges namespace, allowing you to leverage oneDPL parallel algorithms with the standard ranges paradigm. The functionality is implemented for the host and the device execution policies and requires C++20.
+* The oneapi::dpl::experimental::ranges namespace supports integration with oneDPL ranges introducing by oneapi::dpl::experimental::ranges namespace, allowing you to leverage oneDPL parallel algorithms with the range functionality like the Ranges Library from C++20 standard. The functionality is implemented for the device execution policies only and requires C++17.
+
+.. Note::
+
+  The use of the oneapi::dpl::ext::ranges requires C++20 and the C++ standard libraries coming with GCC 10 (or higher) or Clang 10 (or higher).
+  The use of the oneapi::dpl::experimental::ranges requires C++17 and the C++ standard libraries coming with GCC 8.1 (or higher) or Clang 7 (or higher).
+
+
+Range-Based API (The standard C++ Ranges Library)
+-------------------------------------------------
+
+The following C++ standard random access adaptors and factories are supported with the oneDPL parallel algorithms:
+
+* ``std::ranges::views::all``: A range adaptor object that returns a view that includes all elements of its range argument.
+* ``std::ranges::iota_view``: A range factory that generates a sequence of N elements, which starts from an initial value and ends by final N-1.
+* ``std::ranges::single_view``: A view that contains exactly one element of a specified value.
+* ``std::ranges::subrange``: A utility that combines together an iterator and a sentinel into a single object that models a view.
+* ``std::ranges::transform_view``: A range adapter that represents a view of a underlying sequence after applying a transformation to each element.
+* ``std::ranges::reverse_view``: A range adapter that produces a reversed sequence of elements provided by another view.
+* ``std::ranges::take_view``: A range adapter that produces a view of the first N elements from another view.
+* ``std::ranges::drop_view``: A range adapter that produces a view excluding the first N elements from another view.
+
+The following algorithms are available in the oneapi::dpl::ext::ranges namespace to use with the mentioned standard ranges:
+
+* ``for_each``
+* ``transform``
```

Comment:
There are difference in the signature of the parallel and serial algorithms. Could you mention them? This table may be suitable for that.

Reply:
It will be mentioned in the oneDPL specification, and complete signatures will be provided there. Once it's published, we can refer to it from the library guide.